src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
That adds way too many bytes, makes it 513.. not worth it, the whole thing is already unreadable.. hence the comments. I would say no function just minify for the build target.